### PR TITLE
Unity server

### PR DESCRIPTION
**Pull request by RafaelHiguera**

### CODE REVIEW

File: UnityServer/functions/src/controller/userController.ts [hunk 1]
```diff
@@ -0,0 +1,145 @@
+import * as functions from 'firebase-functions';
+import * as admin from 'firebase-admin';
```
Comment (Qing451800) [inline UnityServer/functions/src/controller/userController.ts:2]: License header

File: UnityServer/functions/src/controller/userController.ts [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+      packageName: packageName,
+      // The inapp product SKU (for example, 'com.some.thing.inapp1').
+      productId: productId,
+      // The token provided to the user's device when the inapp product was
+      // purchased.
+      token: purchaseToken,
+    });
+
+    if (result.status === 200) {
+      console.log("200");
```
Comment (Qing451800) [inline UnityServer/functions/src/controller/userController.ts:113]: delete or log a more meaningful message

File: UnityServer/functions/src/controller/userController.ts [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+      // example, 'com.some.thing').
+      packageName: packageName,
+      // The inapp product SKU (for example, 'com.some.thing.inapp1').
+      subscriptionId: productId,
+      // The token provided to the user's device when the inapp product was
+      // purchased.
+      token: purchaseToken,
+    });
+
+    if (result.status === 200) {
```
Comment (Qing451800) [inline UnityServer/functions/src/controller/userController.ts:137]: define 200 as a private static int.

File: UnityServer/functions/src/controller/userController.ts (hunk 2 above)
Comment (RafaelHiguera) [reply UnityServer/functions/src/controller/userController.ts:113]: Yep, will do 👍

File: UnityServer/functions/src/controller/userController.ts (hunk 3 above)
Comment (RafaelHiguera) [reply UnityServer/functions/src/controller/userController.ts:137]: Good idea.

File: UnityServer/functions/src/controller/userController.ts (hunk 1 above)
Comment (RafaelHiguera) [reply UnityServer/functions/src/controller/userController.ts:2]: 👍

**Pull request closed without merging**